eventuate-tram-consumer-jdbc-micronaut/src/main/java/io/eventuate/tram/consumer/jdbc/micronaut/EventuateMicronautTransactionalNoopDuplicateMessageDetector.java

```diff
@@ -0,0 +1,24 @@
+package io.eventuate.tram.consumer.jdbc.micronaut;
+
+import io.eventuate.common.jdbc.micronaut.EventuateMicronautTransactionManagement;
+import io.eventuate.tram.consumer.common.DuplicateMessageDetector;
+import io.eventuate.tram.consumer.common.SubscriberIdAndMessage;
+
+public class EventuateMicronautTransactionalNoopDuplicateMessageDetector implements DuplicateMessageDetector {
```

Comment:
Look at the original version. It has nothing to do with Spring. And, this should have nothing to do with Micronaut. See comment below. The same version should work for both.

Reply:
Chris, if this: https://github.com/dartartem/eventuate-tram-core/blob/master/eventuate-tram-consumer-jdbc/src/main/java/io/eventuate/tram/consumer/jdbc/TransactionalNoopDuplicateMessageDetector.java is original version, it uses spring transaction template. I do not understand.